dist/index.js

```diff
@@ -30,16 +30,19 @@ class SpidPassportBuilder {
     /**
```

Comment:
plase remove the dist folder from this repo. we will publish the package on npm before using it.